factcast-store/src/main/resources/db/changelog/factcast/issue2645/create_published_versions_trigger.sql

```diff
@@ -0,0 +1,34 @@
+-- on fact insert we're saving the type and version of the fact to
+-- prevent extensive queries when enumerating them later.
+
+CREATE TABLE IF NOT EXISTS publishedschemaversions
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS published_schema_versions
```
for readability & consistency

Reply:
would need to be adapted of course